druid-shell/src/keyboard.rs

```diff
@@ -256,6 +256,72 @@ impl KeyCode {
             _ => false,
         }
     }
+
+    pub fn to_char(self) -> Option<char> {
+        use KeyCode::*;
+        match self {
+            KeyA => Some('a'),
```

Comment:
Unfortunately I'm pretty sure this mapping will only be correct on an ANSI keyboard, using american english or compatible; it will not work with alternate keyboard layouts, or alternate languages. We should probably document this better; `KeyCode` here identifies a particular physical key, but that has no guaranteed relationship with the text  produced when that key is pressed.

Reply:
Well. on windows it seems you are converting virtual key codes (i.e. wparam) into `KeyCode` which is hardware independent (as opposed to scancodes in lparam which are the raw inputs sent from the keyboard). 

So it should be fine? I can't really test this as I only have American English keyboards.

Reply:
Never mind, going to research this some more.

Reply:
yea our idea is that `KeyCode` is a platform-independent equivalent to virtual key codes. It is hardware independent, but it is not language independent; for instance you can in settings change your keyboard to greek, without the `KeyCode`s changing.

Good reading for the windows side: https://docs.microsoft.com/en-us/windows/win32/inputdev/about-keyboard-input